.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.11.3-stretch-browsers
+        env:
+          - DISPLAY=:99
+    steps:
+      - checkout
+      - run:
+          command: |
+            sudo apt-get install -y python-setuptools python-dev build-essential xvfb
+            sudo easy_install pip
+            sudo pip install mozdownload mozinstall
+            mozdownload --version 56.0 --destination firefox.tar.bz2
```

Comment:
Should this be updated? Looks like content-server uses 58.

Reply:
ok